scout/run.py

```diff
@@ -5174,16 +5168,25 @@ def main(opts: argparse.NameSpace):  # noqa: F821
     # Instantiate useful variables object
     handyvars = UsefulVars(handyfiles)
 
-    # If a user desires trimmed down results, collect information about whether
-    # they want to restrict to certain years of focus and finalize that year list
-    if opts.trim_results:
+    # User desires trimmed down variable reporting
+    if opts.trim_vars:
         trim_out = True
-        if opts.trim_results == "all_yrs":
-            trim_yrs = False
-        else:
-            trim_yrs = gen_trim_yrs(opts.trim_results, handyvars.aeo_years)
     else:
-        trim_out, trim_yrs = (False for n in range(2))
+        trim_out = False
+    # User desires trimmed down year intervals
+    if opts.change_yr_interval not in [None, 1]:
+        # Check length of AEO years
+        aeo_len = len(handyvars.aeo_years)
+        # Ensure length of year reporting interval doesnt exceed the full length of time horizon
+        if opts.change_yr_interval > aeo_len:
+            opts.change_yr_interval = aeo_len
```

Comment:
What is the desired behavior here? If selecting years based on `if year % yr_interval == 0:`, then setting to aeo_len could output 2025 and 2028 (divisible by 26), which is probably not what anyone wants. should you instead output just the first year and the last year? That could be done by not calling `gen_trim_yrs` if this is true, and instead just setting `trim_yrs` directly